grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -400,13 +402,15 @@ class IncidentPage extends React.Component<IncidentPageProps, IncidentPageState>
                 onSilence: this.getSilenceClickHandler(incident),
                 onUnsilence: this.getUnsilenceClickHandler(incident),
               })}
-              <PluginBridge plugin={SupportedPlugin.Incident}>
-                <a href={incident.declare_incident_link} target="_blank" rel="noreferrer">
-                  <Button variant="secondary" size="md" icon="fire">
-                    Declare incident
-                  </Button>
-                </a>
-              </PluginBridge>
+              {!incident.has_a_related_grafana_incident && (
+                <PluginBridge plugin={SupportedPlugin.Incident}>
+                  <a href={incident.declare_incident_link} target="_blank" rel="noreferrer">
+                    <Button variant="secondary" size="md" icon="fire">
+                      Declare incident
+                    </Button>
+                  </a>
+                </PluginBridge>
+              )}
```

Comment:
hide "Declare Incident" button if this Alert Group was generated from Grafana Incident (via a Direct Page)